webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -735,30 +694,30 @@ protected function validateAndUpdateContest(string $entityType, ?string $eventId
 
         // We need to convert the freeze to a value from the start instead of
         // the end so perform some regex magic.
-        $duration     = $data['duration'];
-        $freeze       = $data['scoreboard_freeze_duration'];
+        $duration = $data->duration;
+        $freeze = $data->scoreboardFreezeDuration;
         $reltimeRegex = '/^(-)?(\d+):(\d{2}):(\d{2})(?:\.(\d{3}))?$/';
         preg_match($reltimeRegex, $duration, $durationData);
 
         $durationNegative = ($durationData[1] === '-');
-        $fullDuration     = $durationNegative ? $duration : ('+' . $duration);
+        $fullDuration = $durationNegative ? $duration : ('+' . $duration);
 
         if ($freeze !== null) {
             preg_match($reltimeRegex, $freeze, $freezeData);
-            $freezeNegative       = ($freezeData[1] === '-');
-            $freezeHourModifier   = $freezeNegative ? -1 : 1;
-            $freezeInSeconds      = $freezeHourModifier * (int)$freezeData[2] * 3600
-                                    + 60 * (int)$freezeData[3]
-                                    + (double)sprintf('%d.%03d', $freezeData[4], $freezeData[5]);
+            $freezeNegative = ($freezeData[1] === '-');
+            $freezeHourModifier = $freezeNegative ? -1 : 1;
+            $freezeInSeconds = $freezeHourModifier * (int)$freezeData[2] * 3600
+                + 60 * (int)$freezeData[3]
+                + (double)sprintf('%d.%03d', $freezeData[4], $freezeData[5]);
             $durationHourModifier = $durationNegative ? -1 : 1;
-            $durationInSeconds    = $durationHourModifier * (int)$durationData[2] * 3600
-                                    + 60 * (int)$durationData[3]
-                                    + (double)sprintf('%d.%03d', $durationData[4], $durationData[5]);
-            $freezeStartSeconds   = $durationInSeconds - $freezeInSeconds;
-            $freezeHour           = floor($freezeStartSeconds / 3600);
-            $freezeMinutes        = floor(($freezeStartSeconds % 3600) / 60);
-            $freezeSeconds        = floor(($freezeStartSeconds % 60) / 60);
-            $freezeMilliseconds   = $freezeStartSeconds - floor($freezeStartSeconds);
+            $durationInSeconds = $durationHourModifier * (int)$durationData[2] * 3600
+                + 60 * (int)$durationData[3]
+                + (double)sprintf('%d.%03d', $durationData[4], $durationData[5]);
+            $freezeStartSeconds = $durationInSeconds - $freezeInSeconds;
+            $freezeHour = floor($freezeStartSeconds / 3600);
+            $freezeMinutes = floor(($freezeStartSeconds % 3600) / 60);
+            $freezeSeconds = floor(($freezeStartSeconds % 60) / 60);
+            $freezeMilliseconds = $freezeStartSeconds - floor($freezeStartSeconds);
```

Comment:
I disagree with this kind of removal of alignment spaces. It makes the code less readable.

Reply:
My bad, this is what the code style currently says. I will undo these changes.

Reply:
I don't know exactly what the current code style says, but I strongly think that automatic reformatting of alignment like this should be removed from it. Code formatting is a means to an end (better readability of code) and not a goal in itself.

Reply:
I believe the problem is it wants some alignment and you can’t say “do whatever there was” but I hope I’m wrong since that would save me time